core/src/main/java/com/graphhopper/routing/util/TraversalMode.java

```diff
@@ -37,25 +37,16 @@
     /**
      * The simplest traversal mode but without turn restrictions or cost support.
      */
-    NODE_BASED(false, false),
+    NODE_BASED(false),
     /**
-     * The bidirectional edged-based traversal mode with turn restriction and cost support. Without
-     * u-turn support. 2 times slower than node based.
+     * The edged-based traversal mode with turn restriction and cost support. 2 times slower than node based.
      */
-    EDGE_BASED_2DIR(true, false),
-    /**
-     * Not recommended as it leads to strange routes that outsmart the turn costs. The most feature
-     * rich edge-based traversal mode with turn restriction and cost support, including u-turns. 4
-     * times slower than node based.
-     */
-    EDGE_BASED_2DIR_UTURN(true, true);
+    EDGE_BASED(true);
```

Comment:
Should we keep `EDGE_BASED_2DIR` as alias for `EDGE_BASED` for better compatibility with user code ?

Reply:
I'm for removing (as it is easy to do on the user code side)

Reply:
ok